meshtastic/node_runtime/seturl/cache.py

```
@@ -0,0 +1,172 @@
+if TYPE_CHECKING:
+    from meshtastic.node import Node
+
+logger = logging.getLogger(__name__)
+
+_ERR_CONFIG_OR_CHANNELS_NOT_LOADED = "Config or channels not loaded"
+
+
+class _SetUrlCacheManager:
+    """Owns local cache updates, invalidation, and restoration for setURL transactions."""
+
+    def __init__(self, node: "Node") -> None:
```

Ruff (0.15.7)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)